habitat-lab/habitat/datasets/rearrange/navmesh_utils.py

```diff
@@ -463,11 +465,12 @@ def act(self, trans: mn.Matrix4, vel: List[float]) -> mn.Matrix4:
 
         :return: The updated agent transformation matrix.
         """
+
         linear_velocity = vel[0]
         angular_velocity = vel[1]
         # Map velocity actions
         self.vel_control.linear_velocity = mn.Vector3(
-            [0.0, 0.0, -linear_velocity]
+            [linear_velocity, 0.0, 0.0]
```

Comment:
hi alex, do you know why we do this at first place?

Reply:
why [0.0, 0.0, -linear_velocity]?

Reply:
Originally, this was alligned with Habitat's coordinate system. Typical Agent "forward" direction in local space is -Z.